ValveResourceFormat/Resource/ResourceTypes/Model.cs

```diff
@@ -172,5 +176,141 @@ public IEnumerable<bool> GetActiveMeshMaskForGroup(string groupName)
                 return meshGroupMasks.Select(_ => false);
             }
         }
+
+        public void GenerateMorphTargets(IFileLoader fileLoader, ModelRoot exportModel)
```

Comment:
Let's not put glTF stuff in resource types.

Put it in the existing gltf file for now (we should split it into multiple files later).

Reply:
ok